### ISSUE TITLE
Login issue - KeyError: 'authenticated'

### ISSUE THREAD

**Issue by samsepiol59** [instaloader/instaloader#472]
**Describe the bug**
Can't login

**To Reproduce**
Try and login with cli

**Expected behavior**
It should login

**Error messages and tracebacks**
```
samsepiol@samsepiol:~$ /usr/local/bin/instaloader :stories --no-captions --no-metadata-json --dirname-pattern /home/samsepiol/datahoarder/instagram/{profile} --filename-pattern "{profile} - {date_utc} - {shortcode}" --login=mylogin --password=mypasswd
Session file does not exist yet - Logging in.
Traceback (most recent call last):
  File "/usr/local/bin/instaloader", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/instaloader/__main__.py", line 414, in main
    storyitem_filter_str=args.storyitem_filter)
  File "/usr/local/lib/python3.6/dist-packages/instaloader/__main__.py", line 89, in _main
    instaloader.login(username, password)
  File "/usr/local/lib/python3.6/dist-packages/instaloader/instaloader.py", line 429, in login
    self.context.login(user, passwd)
  File "/usr/local/lib/python3.6/dist-packages/instaloader/instaloadercontext.py", line 223, in login
    if not resp_json['authenticated']:
KeyError: 'authenticated'
samsepiol@samsepiol:~$
```

**Instaloader version**
4.2.10

**Additional context**
I used it daily for the past two weeks if not more, probably two runs per day

**Comment by dausruddin** [comment]
I am having the same issue. Probably same as this one? https://github.com/instaloader/instaloader/issues/383

**Comment by dausruddin** [comment]
Turned out Instagram block my IP. I cannot login via browser.

**Comment by shawnbarton** [comment]
I have this issue and can confirm it seems that Instagram is blocking either my IP or provider ASN.

``
wget https://www.instagram.com/test_profile/
Saving to: ‘index.html’
cat index.html
{"errors": {"error": ["Sorry, there was a problem with your request."]}, "status": "ok", "error_type": "generic_request_error"}
``

**Comment by aandergr** [comment]
We implemented catching this error and printing an appropriate message, that this might indicate a blocked IP, with one of our latest commits, and it will be included in the upcoming version, Instaloader 4.4.
